### PR TITLE
yanic: use domain_code instead of site_code 

### PR DESCRIPTION
**Pull request by T0biii**
fixes maybe #151

### CODE REVIEW

**Comment by grische** [comment]
@T0biii this does not seem to have any effect

**Comment by T0biii** [comment]
more Infos in chat i guess?
https://chat.ffmuc.net/freifunk/pl/auhwc9um8if75fgtt6kdxqtqpa

File: yanic/yanic.conf.tmpl [hunk 1]
```diff
@@ -59,7 +59,7 @@ path = "/srv/yanic/meshviewer-ffdon.json"
 
 [nodes.output.meshviewer-ffrgb.filter]
 sites = ["ffdon"]
-
+domain_as_site = true
```
Comment (DasSkelett) [inline yanic/yanic.conf.tmpl:62]: This is how I got it to work now, and with the `[respondd.sites.ffdon]` section above being removed:
```suggestion
domain_as_site = true
sites = [{% for domain in sites["ffdon"] %}"ffdon_{{ domain }}", {% endfor %}]
```

**Comment by T0biii** [comment]
added the suggested changes

**Review by DasSkelett — APPROVED** [review]

**Pull request merged**